src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: PreparedTodo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => (
+  <div className="to_do__info">
+    <h2 className="to_do__info_title">
+      The goal:
+      {' '}
+      {todo.title}
+    </h2>
+
+    <UserInfo user={todo.user} />
+
+    {todo.completed ? (
+      <span className="to_do__info_complete--done">
+        Completed
+      </span>
+    ) : (
+      <span className="to_do__info_complete--not_done">
+        Not completed
+      </span>
+    )}
```

Comment:
а лучше используй https://www.npmjs.com/package/classnames

```suggestion
    {todo.completed 
      ? (
          <span className="to_do__info_complete--done">
            Completed
          </span>
        ) 
      : (
          <span className="to_do__info_complete--not_done">
            Not completed
          </span>
       )}
```

Reply:
Не понимаю как его тут использовать.
У меня же меняется не только класс, но и текст в теге.
Чем тут поможет classnames?

Reply:
для классов используешь classnames,
а для текста внутри тега используешь conditional rendering